cpp/src/IceGrid/Makefile.mk

```diff
@@ -5,34 +5,34 @@ local_node_srcs         = Activator.cpp \
                           NodeI.cpp \
                           NodeSessionManager.cpp \
                           ServerAdapterI.cpp \
-                          ServerI.cpp \
-                          XMLParser.cpp \
```

Comment:
These 3 files are about the registry so it's odd to list them here. 

Keep in mind that icegridnode always includes almost all icegridregistry source files to the collocated registry misfeature.